pkg/commands/handler.go

```diff
@@ -0,0 +1,214 @@
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/fly-examples/postgres-ha/pkg/flypg"
+	"github.com/fly-examples/postgres-ha/pkg/render"
+	"github.com/jackc/pgx/v4"
+)
+
+const Port = 5600
+
+func StartCommandsHandler() {
+
+	http.HandleFunc("/commands/users", handleListUsers)
+	http.HandleFunc("/commands/users/create", handleCreateUser)
+	http.HandleFunc("/commands/users/delete", handleDeleteUser)
+
+	http.HandleFunc("/commands/users/grant", handleGrantAccess)
```

Comment:
You can ditch the grant/revoke handlers, these operations should be handled by user create/delete.